salt/kubelet/kubelet.jinja

```diff
@@ -32,5 +44,6 @@ KUBELET_ARGS="\
     --cloud-provider=aws \
 {% endif -%}
     {{ pillar['components']['kubelet']['args'] }} \
+    --node-labels={{ node_labels|join(',') }} \
```

Comment:
Providing `--node-labels=` is harmless (as a matter of fact, that's the default)

As far as changing the labels, it's more complicated than just the kubectl call. These labels affect scheduling of containers. Changing the labels will not cause the containers to be rescheduled.